### PR TITLE
[DOC] fix inconsistent citation style

### PR DESCRIPTION
**Pull request by areeshatariq**
We should stick to named references rather than numbered for consistent citation style.

### CODE REVIEW

File: doc/interfaces/bundle_segmentation_flow.rst [hunk 1]
```diff
@@ -106,26 +106,26 @@ original space, run following commands::
 For more information about each command line, please visit DIPY website `<https://dipy.org/>`_ .
 
 If you are using any of these commands please be sure to cite the relevant papers and
-DIPY [4]_.
+DIPY [Garyfallidis14s]_.
```
Comment (jhlegarreta) [inline doc/interfaces/bundle_segmentation_flow.rst:109]: Typo: Garyfallidis14s : Garyfallidis14

File: doc/interfaces/gibbs_unringing_flow.rst [hunk 2]
```diff
@@ -69,10 +69,10 @@ the Gibbs unringing effect.
...
-       2015. Gibbs Ringing in Diffusion MRI. Magn Reson Med 76(1): 301-314.
-       https://doi.org/10.1002/mrm.25866
-.. [2] Perrone, D., Aelterman, J., Pižurica, A., Jeurissen, B., Philips, W.,
-       Leemans A., 2015. The effect of Gibbs ringing artifacts on measures
-       derived from diffusion MRI. Neuroimage 120, 441-455.
-       https://doi.org/10.1016/j.neuroimage.2015.06.068.
\ No newline at end of file
+.. [Veraart15] Veraart, J., Fieremans, E., Jelescu, I.O., Knoll, F., Novikov, D.S.,
+               2015. Gibbs Ringing in Diffusion MRI. Magn Reson Med 76(1): 301-314.
+               https://doi.org/10.1002/mrm.25866
```
Comment (jhlegarreta) [inline doc/interfaces/gibbs_unringing_flow.rst:74]: Why are the whitespaces introduced here but not in the other references? I think we should choose and stick one style if both are rendered correctly.

Comment (areeshatariq) [reply doc/interfaces/gibbs_unringing_flow.rst]: Agreed! Addressed in 54907a3

File: doc/interfaces/bundle_segmentation_flow.rst (hunk 1 above)
Comment (areeshatariq) [reply doc/interfaces/bundle_segmentation_flow.rst]: Addressed in 54907a3

**Review by jhlegarreta — APPROVED** [review]
Looks good to me. Thanks for doing this.

**Comment by jhlegarreta** [comment]
@skoudoro not sure what your concern was about the change (I have not tested it).

**Comment by skoudoro** [comment]
thanks @areeshatariq

**Pull request merged**